models/repo_language_stats.go

```diff
@@ -144,3 +144,36 @@ func (repo *Repository) UpdateLanguageStats(commitID string, stats map[string]fl
 
 	return sess.Commit()
 }
+
+func CopyLanguageStat(oldRepo,Repo *Repository) error {
```

Comment:
maybe `originalRepo` and `dstRepo` is better names.

Reply:
Thanks your advice.The new names are more clear. I will use the new names to replace the old.